tests/starwars/test_query.py

```diff
@@ -343,12 +343,12 @@ def test_check_type_of_luke(client):
 
 def test_parse_error(client):
     result = None
-    with pytest.raises(Exception) as excinfo:
+    with pytest.raises(Exception) as excInfo:
```

Comment:
You're right, I took a quick look at pytest docs and found the variable being written as `exec_info` instead of `excInfo`. Not sure why I used the latter convention.

Reply:
It's just PEP8 convention.